index.html

```diff
@@ -59,7 +59,7 @@ <h2>Contact</h2>
       <section class="skills">
         <h2>Skills</h2>
         <ul>
-          <li>Web Development:<code>HTML,</code><code>CSS,</code><code>JavaScript,</code><code>Java,</code></li>
+          <li>Web Development:<abbr title="Hypertext Markup Language">HTML</abbr>,<abbr title="Cascading Style Sheets">CSS</abbr>,<abbr title="JavaScript">JS</abbr> &amp;Java,</li>
           <li>Languages: English, Spanish, And French</li>
```

Comment:
you need to use for colons (:) entities characters. Revise this in all your html. There are more than one...